core/system/src/mpu/vmpu_freescale_k64.c

```diff
@@ -110,51 +106,48 @@ void vmpu_sys_mux_handler(uint32_t lr)
     }
 }
 
-void vmpu_acl_add(uint8_t box_id, void* start, uint32_t size, UvisorBoxAcl acl)
+void vmpu_acl_add(uint8_t box_id, void *start, uint32_t size, UvisorBoxAcl acl)
 {
     int res;
 
 #ifndef NDEBUG
     const MemMap *map;
-#endif/*NDEBUG*/
+#endif /* NDEBUG */
 
     /* check for maximum box ID */
-    if(box_id>=UVISOR_MAX_BOXES)
+    if (box_id >= UVISOR_MAX_BOXES)
         HALT_ERROR(SANITY_CHECK_FAILED, "box ID out of range (%i)\n", box_id);
 
     /* check for alignment to 32 bytes */
-    if(((uint32_t)start) & 0x1F)
+    if (((uint32_t) start) & 0x1F)
         HALT_ERROR(SANITY_CHECK_FAILED, "ACL start address is not aligned [0x%08X]\n", start);
 
     /* round ACLs if needed */
-    if(acl & UVISOR_TACL_SIZE_ROUND_DOWN)
+    if (acl & UVISOR_TACL_SIZE_ROUND_DOWN)
         size = UVISOR_REGION_ROUND_DOWN(size);
-    else
-        if(acl & UVISOR_TACL_SIZE_ROUND_UP)
-            size = UVISOR_REGION_ROUND_UP(size);
+    else if (acl & UVISOR_TACL_SIZE_ROUND_UP)
+        size = UVISOR_REGION_ROUND_UP(size);
 
     DPRINTF("\t@0x%08X size=%06i acl=0x%04X [%s]\n", start, size, acl,
-        ((map = memory_map_name((uint32_t)start))!=NULL) ? map->name : "unknown"
-    );
+            ((map = memory_map_name((uint32_t) start)) != NULL) ?
+                map->name : "unknown");
```

Comment:
I would merge lines 132-133 here, since they are still below 120 char in the column and make the ternary operator more readable in my opinion